ansible/playbooks/apache.yml

```
@@ -0,0 +1,27 @@
+- hosts: linux solaris
+  become: yes
+  become_user: root
+  roles:
+    - ../roles/apache
+  vars:
+    apache_hosts:
+      - port: 1514
+        protocol: tcp
+        api_port: 55000
+        api_proto: 'http'
+        api_user: ansible
+        max_retries: 5
+        retry_interval: 5
+
+- hosts: windows macos
+  roles:
+    - ../roles/apache
+  vars:
+    apache_hosts:
+      - port: 1514
+        protocol: tcp
+        api_port: 55000
+        api_proto: 'http'
+        api_user: ansible
+        max_retries: 5
+        retry_interval: 5
```

Use apache as the host group instead of the operative system. In order to split the use of `become/become_user` you could use a conditional.
```suggestion
- hosts: apache
  tasks:
  - name: Install apache block
    - block:
       - name: Install apache server in Linux systems
         become: yes
         become_user: root
         include_vars: apache_configuration_template.yml 
          import_role: 
            name: ../roles/apache
          when os == 'linux' or os == 'solaris-11' 
       - name: Install apache server in Windows and macOS hosts
         include_vars: apache_configuration_template.yml 
          import_role: 
            name: ../roles/apache
          when os == 'windows' or os == 'macos' 
```